libyul/Object.h

```diff
@@ -68,7 +68,15 @@ struct Object: ObjectNode
 
 	/// @returns the set of names of data objects accessible from within the code of
 	/// this object.
-	std::set<YulString> dataNames() const;
+	std::set<YulString> qualifiedDataNames() const;
+
+	/// @returns tuple consisting of
+	/// 	bool true if _qualifiedName is referring to data
+	/// 	vector of subIDs if possible to reach subobject with @_qualifiedName, empty vector otherwise
+	/// for "B.C" should return vector of two values if success (subId of B and subId of C in B)
+	/// in object "A" if called for "A.B" will return only one value (subId for B)
+	/// will return empty vector for @_qualifiedName that equals to object name
```

Comment:
I think that the syntax for highlighting identifiers in doxygen docstrings is `@a _qualifiedName`.

Also, this is a bit hard to understand without capitals and full stops. Did I interpret it correctly?
```suggestion
	/// @returns tuple consisting of
	///     - bool true if @a _qualifiedName is referring to data,
	///     - vector of subIDs if possible to reach subobject with @a _qualifiedName, empty vector otherwise
	/// For "B.C" should return vector of two values if success (subId of B and subId of C in B).
	/// In object "A" if called for "A.B" will return only one value (subId for B)
	/// will return empty vector for @a _qualifiedName that equals to object name.
```

I'd also suggest including an example. I tried reading this before I read the code of the function but I didn't really understand it. Maybe the path part could be illustrated with something like this:
``` c++
/// A1 { B2 { C3 D3 }, E2 { F3 { G4 K3 H4 { I5 } } } }
/// pathToSubObject("A1.E2.F3.H4") == {1, 0, 2}
/// pathToSubObject("E2.F3.H4") == {1, 0, 2}
/// pathToSubObject("A1") == {}
```
Not sure how to succintly include `data` in it though.